.travis.yml

```diff
@@ -57,9 +66,23 @@ before_install:
 - sudo apt-get update
 - sudo apt-get install -y --force-yes r-base
 - R --version
-#install gcc
-- sudo apt-get install -y --force-yes  gcc
-- gcc --version
+#Install lcov and MPICH
```

Comment:
Document in a comment which dependencies are for TileDB, so that we can remove them later if we get our wish of a statically-linked binary.